Dockerfile

```
@@ -1,2 +1,2 @@
-FROM semtech/mu-javascript-template:1.6.0
+FROM semtech/mu-javascript-template:1.6.0:feature-node-18
```

woops something went wrong here I think, this should be 
```suggestion
FROM semtech/mu-javascript-template:feature-node-18
```